### PR TITLE
Feature/navigation two

### PR DESCRIPTION
**Pull request by rs-mgb**
Still some low-prio TODO inside.

### CODE REVIEW

**Review by Weedshaker — APPROVED** [review]
Fantastic Work!!! 1158 Lines of sweat and pure brain power :-)

**Review by Weedshaker — APPROVED** [review]

**Review by Weedshaker — DISMISSED** [review]

File: src/es/components/organisms/header/default-/default-.css [hunk 1]
```diff
@@ -38,7 +38,8 @@
   --header-default-logo-width: var(--header-default-logo-width-custom, 90px);
   --header-default-m-navigation-background-color-mobile: var(--background-color);
   --header-default-m-navigation-height-mobile: calc(100vh - var(--header-default-m-navigation-top-mobile));
-  --header-default-m-navigation-height-open-mobile: var(--header-default-m-navigation-height-mobile);
```
Comment (Weedshaker) [inline src/es/components/organisms/header/default-/default-.css:41]: Is this working without side effect on the existing navigation? Did you test it with mpro or alna?

**Review by Weedshaker — CHANGES_REQUESTED** [review]
Please, make sure we don't have any backward compatibility issues... THX

**Review by Weedshaker — DISMISSED** [review]

**Review by Weedshaker — DISMISSED** [review]

**Review by Weedshaker — APPROVED** [review]
fantastic work!

File: src/es/components/organisms/header/default-/default-.css (hunk 1 above)
Comment (rs-mgb) [reply src/es/components/organisms/header/default-/default-.css:41]: @Weedshaker Accidentally removed, other components need this variable, i added back.

**Review by Weedshaker — APPROVED** [review]

**Pull request merged**